kernel/configs/x86_64_kata_kvm_4.14.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.14.49 Kernel Configuration
+# Linux/x86 4.14.67 Kernel Configuration
```

Comment:
I see a 4.14.75 is out... maybe that just happened, but should we consider that version instead?
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/Makefile?h=linux-4.14.y